open-craft/edx-platform
# [BD-14] [SE-2932] Add a license field to libraries.

This Pull request adds a simple 'license' field with a handful of supported values to the backend. It is not intended to be as fully functional as the current course licensing schema, but is intended to provide for the most common cases until business needs require a more complex implementation.

**Discussions**: https://docs.google.com/document/d/1hRrSmWWlGHhi-bh9AIlpSuLtJFnn2GpavNe8OQxthbE/

**Dependencies**: https://github.com/edx/edx-platform/pull/24861

**Merge deadline**: This is the review copy of this pull request, and is not to be merged.

**Testing instructions**:

1. Standard setup as the other PRs have been so far (check https://github.com/edx/edx-platform/pull/24861 for initial setup).
2. Run migrations.
3. Run tests.

**Reviewers**
